pjmedia/src/pjmedia/vid_stream.c

```diff
@@ -1902,16 +1916,20 @@ PJ_DEF(pj_status_t) pjmedia_vid_stream_create(
     stream->dec_frame.buf = pj_pool_alloc(pool, stream->dec_max_size);
 
     /* Init jitter buffer parameters: */
-    frm_ptime	    = 1000 * vfd_enc->fps.denum / vfd_enc->fps.num;
+    frm_ptime	    = 1000 * vfd_dec->fps.denum / vfd_dec->fps.num;
     chunks_per_frm  = stream->frame_size / PJMEDIA_MAX_MRU;
     if (chunks_per_frm < MIN_CHUNKS_PER_FRM)
 	chunks_per_frm = MIN_CHUNKS_PER_FRM;
 
     /* JB max count, default 500ms */
-    if (info->jb_max >= frm_ptime)
+    if (info->jb_max >= frm_ptime) {
 	jb_max	    = info->jb_max * chunks_per_frm / frm_ptime;
-    else
+	stream->dec_max_delay = info->codec_info.clock_rate * info->jb_max /
+				1000;
+    } else {
 	jb_max	    = 500 * chunks_per_frm / frm_ptime;
+	stream->dec_max_delay = info->codec_info.clock_rate * 500 / 1000;
+    }
```

Comment:
Looks like audio also uses 500ms as default jb_max, perhaps we need to change the pjsua docs too (which specifies 360ms):
```
    /**
     * Set maximum delay that can be accomodated by the jitter buffer msec.
     *
     * Default: -1 (to use default stream settings, currently 360 msec)
     */
    int			jb_max;
```